src/main/com/aca/homework/week10/object/factory/ServletRequestObjectFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.aca.homework.week10.object.factory;
+
+public class ServletRequestObjectFactory<T> implements ObjectFactory<T> {
```

Comment:
ServletRequestObjectFactory is a concrete class of ObjectFactory<ServletRequest> type